contracts/sol6/KyberNetwork.sol

```diff
@@ -1133,13 +1134,14 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
         }
 
         // reverse calculation, because we are working backwards
-        tradeData.tradeWei =
+        uint256 newTradeWei =
             (weiAfterFees * BPS * BPS) /
```

Comment:
maybe we should still improve calculation here.
like mul with BPS ** 4

anyway its better to mul with constant value. and avoid const * const
seems the compiler doesn't do it in compilation time...

Reply:
My concern is that multiplying with that large a number might cause overflow problems

Reply:
I guess it depends what's the maximum theoretical value of `calcTradeSrcAmount`?

Reply:
OK
seems to me we should not update this value.
worst case is a few wei error.
we can keep existing numbers.